### PR TITLE
Wordsmith docs overview page

### PR DESCRIPTION
**Pull request by adidahiya**

### CODE REVIEW

**Comment by cmslewis** [comment]
Let's edit the repo description in Github too?

image

**Comment by adidahiya** [comment]
@cmslewis updated the landing page. I'll update the repo description when this merges.

File: packages/docs/src/styleguide.md [hunk 1]
```diff
@@ -36,8 +38,8 @@ relevant sections in the sidebar for details about the styling & component APIs
 ### TypeScript
 
 Blueprint is written in [TypeScript](https://www.typescriptlang.org/), a statically typed superset
-of JavaScript that compiles to plain JavaScript. Many of the code samples throughout this site and
-all interactive examples are also written in TypeScript. TypeScript code looks exactly like ES2015
+of JavaScript that compiles to plain JavaScript. All the code samples throughout this site and
+all interactive examples are also written in TypeScript. TypeScript code looks exactly like ES2015+
```
Comment (giladgray) [inline packages/docs/src/styleguide.md]: whats the + for?

File: packages/docs/src/styleguide.md [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
-Development occurs in [palantir/blueprint](https://github.com/palantir/blueprint).
-Please file issues on the [GitHub issue tracker](https://github.com/palantir/blueprint/issues).
+Development and issue tracking occurs in [github.com/palantir/blueprint](https://github.com/palantir/blueprint).
 
-Releases are tagged and documented [here](https://github.com/palantir/blueprint/releases)
+Use the [__blueprintjs__ tag on Stack Overflow](http://stackoverflow.com/questions/tagged/blueprintjs)
+for support requests.
+
+Releases are tagged and documented [here on Github](https://github.com/palantir/blueprint/releases)
 (more info about releases [on the wiki](https://github.com/palantir/blueprint/wiki/Releases-&-Artifacts)).
```
Comment (giladgray) [inline packages/docs/src/styleguide.md]: pretty sure this wiki page does not exist

File: packages/docs/src/styleguide.md (hunk 1 above)
Comment (adidahiya) [inline packages/docs/src/styleguide.md]: because there's ES2016, ES2017...

File: packages/docs/src/styleguide.md (hunk 2 above)
Comment (adidahiya) [inline packages/docs/src/styleguide.md]: Not part of this PR

File: packages/docs/src/styleguide.md [hunk 3]
```diff
@@ -1,16 +1,18 @@
-Blueprint is a React UI toolkit for the web.
+Blueprint is a React-based UI toolkit for the web.
```
Comment (llorca) [inline packages/docs/src/styleguide.md]: I think it's way cleaner without the verboseness of the "-based" (not really necessary). If you feel strongly, I'm cool with here, but I prefer keeping the tagline everywhere as is for now. Note that changing it in this file only isn't enough, there are other places where it appears on GitHub, the landing, different services... Let's wait for 2.0 please, and we can work out and even better one.

File: packages/docs/src/styleguide.md (hunk 2 above)
Comment (llorca) [inline packages/docs/src/styleguide.md]: It exists internally, I looked at it, seems pretty unnecessary and out-of-date. Let's remove this now?

File: packages/docs/src/styleguide.md [hunk 4]
```diff
@@ -23,9 +25,9 @@ Don't forget to include the main CSS stylesheet too!
 
 ### Beyond core styles
 
-The Blueprint team maintains multiple packages of styles and JavaScript components on NPM (in the scope
-`@blueprintjs`). These are listed in the header under _Releases_. They can be installed with
-similar commands as above. Once you've included any additional package(s) in your application, explore the
+Blueprint is a collection of packages of styles and JavaScript components&mdash;the full list appears in this
```
Comment (llorca) [inline packages/docs/src/styleguide.md]: First read I thought the "full list" referred to the "components" and not the "packages". Suggestion: `the full list appears` -> `those packages appear`?

File: packages/docs/src/styleguide.md (hunk 1 above)
Comment (giladgray) [inline packages/docs/src/styleguide.md]: yeah that's what i thought, just looked like a typo at first

File: packages/docs/src/styleguide.md (hunk 3 above)
Comment (llorca) [inline packages/docs/src/styleguide.md]: Looks like I'm the only one thinking like this, so nevermind.

Can you make sure you caught all instances of that tagline and that it's changed everywhere?

**Pull request merged**